docker-compose.yml

```diff
@@ -52,8 +55,21 @@ services:
     ports:
       - ${PG_PORT}:5432
     volumes:
-      - './postgres:/docker-entrypoint-initdb.d/'
       - 'pg_data:/var/lib/postgresql/data'
+
+  migration:
+    container_name: migration
+    build:
+      context: ./postgres
+      dockerfile: ./Dockerfile
+    depends_on:
+      - db
+    environment:
+      - POSTGRES_HOST=db
+      - POSTGRES_DB=${PG_DB}
+      - POSTGRES_USER=${PG_USER}
+      - POSTGRES_PASSWORD=${PG_PASSWORD}
+
   staging_db:
     container_name: pg_container_testing
     image: postgres

```

Comment:
What's the purpose of this db? (compared to the other `db` service in this file)

Reply:
It was/is an ease of use DB we have to run any database bound tests against, although we already wrap all our tests in transactions so its not entirely required, ill remove it within another PR if need be